### PR TITLE
Fix folder path filter when looking for locales

### LINKED ISSUE #13: Find correct locales folder for the project
Need to review this last change, the grep to filter recently added for the directory works because the '.' means anything in regex.

I can't imagine all regex possibilities, but the package name could add another bug in this script.

My suggestion is to use the -F option of grep to make it works without regex comparison.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #13

### CODE REVIEW

File: qa.cfg [hunk 1]
```diff
@@ -63,7 +63,8 @@ input = inline:
     I18NDOMAIN="${buildout:package-name}"
 
     # find the locales dir
-    LOCALES=`find . -type d | grep -m 1 "locales"`
+    FOLDER_PATH="`echo ${buildout:package-name} | sed -e 's/\./\//'`"
```
Comment (rodfersou) [inline qa.cfg]: For example `brasil.gov.portal` becomes `brasil/gov/portal`

File: templates/rebuild_i18n.sh.in [hunk 2]
```diff
@@ -7,7 +7,8 @@ echo "    https://github.com/collective/buildout.plonetest/issues/11"
 I18NDOMAIN="${buildout:package-name}"
 
 # find the locales dir
-LOCALES=`find . -type d | grep "${buildout:package-name}" | grep -m 1 "locales"`
+FOLDER_PATH="`echo ${buildout:package-name} | sed -e 's/\./\//'`"
```
Comment (rodfersou) [inline templates/rebuild_i18n.sh.in]: For example `brasil.gov.portal` becomes `brasil/gov/portal`

File: qa.cfg [hunk 3]
```diff
@@ -63,7 +63,8 @@ input = inline:
...
+    LOCALES=`find . -type d | grep -F "$FOLDER_PATH" | grep -m 1 "locales"`
```
Comment (rodfersou) [inline qa.cfg]: `grep -F` searches by the exact string instead of parse regex

File: templates/rebuild_i18n.sh.in [hunk 4]
```diff
@@ -7,7 +7,8 @@ echo "    https://github.com/collective/buildout.plonetest/issues/11"
...
+LOCALES=`find . -type d | grep -F "$FOLDER_PATH" | grep -m 1 "locales"`
```
Comment (rodfersou) [inline templates/rebuild_i18n.sh.in]: `grep -F` searches by the exact string instead of parse regex

**Comment by rodfersou** [comment]
@hvelarde could you please review this?

**Comment by rodfersou** [comment]
@tmassman I think you change the wrong file in the last pull request

**Comment by hvelarde** [comment]
I think is easier just to get rid of the quotes:

```bash
LOCALES=`find . -type d | grep ${buildout:package-name} | grep -m 1 "locales"`
```

**Comment by tmassman** [comment]
@rodfersou My pull request was opened on **May 27, 2014**. That was way before @hvelarde changed the i18n part to use the inline template ;)
But you are right, the inline template should be changed as well.

**Comment by rodfersou** [comment]
@tmassman got it!
@hvelarde changed the way you wish!

**Pull request merged**